test/GitcoinGovernor.t.sol

```diff
@@ -301,137 +289,177 @@ contract GitcoinGovernorAlphaPostProposalTest is GitcoinGovernorProposalTestHelp
       governorAlpha.castVote(_newProposalId, true);
     }
     vm.roll(_endBlock + 1);
+
+    if (!isGovernorAlphaAdmin) {
+      vm.expectRevert("Timelock::queueTransaction: Call must come from admin.");
+      governorAlpha.queue(_newProposalId);
+      return;
+    }
+
     governorAlpha.queue(_newProposalId);
     vm.roll(block.number + 1);
     (,, uint256 _eta,,,,,,) = governorAlpha.proposals(_newProposalId);
     vm.warp(_eta + 1);
+
     governorAlpha.execute(_newProposalId);
 
     // Ensure the new proposal is now executed
-    uint8 _state = governorAlpha.state(_newProposalId);
-    assertEq(_state, EXECUTED);
+    assertEq(governorAlpha.state(_newProposalId), EXECUTED);
+  }
+
+  function testFuzz_OldGovernorSendsETHAfterProposalIsDefeated(uint256 _amount, address _receiver)
+    public
+  {
+    _assumeReceiver(_receiver);
+
+    // Counter-intuitively, the Governor must hold the ETH, not the Timelock.
+    // See the deployed GovernorAlpha, line 227:
+    //   https://etherscan.io/address/0xDbD27635A534A3d3169Ef0498beB56Fb9c937489#code
+    // The governor transfers ETH to the Timelock in the process of executing
+    // the proposal. The Timelock then just passes that ETH along.
+    vm.deal(address(governorAlpha), _amount);
+
+    uint256 _receiverETHBalance = _receiver.balance;
+    uint256 _governorETHBalance = address(governorAlpha).balance;
+
+    // Defeat the proposal to upgrade the Governor
+    _defeatUpgradeProposal();
+
+    // Create a new proposal to send the ETH.
+    address[] memory _targets = new address[](1);
+    uint256[] memory _values = new uint256[](1);
+    _targets[0] = _receiver;
+    _values[0] = _amount;
+
+    _queueAndVoteAndExecuteProposalWithAlphaGovernor(
+      _targets,
+      _values,
+      (new string[](1)), // No signature needed for an ETH send.
+      (new bytes[](1)), // No calldata needed for an ETH send.
```

Comment:
Nit, these outer parentheses can be removed. If you prefer keeping them, that's fine too—feel free to resolve this comment yourself.

Reply:
Done in 4f59e02!